udata-tabular-preview/explore.vue

```diff
@@ -74,37 +96,116 @@ export default defineComponent({
   },
   setup(props) {
     const {
-      apifyAndGetData,
-      changePage,
-      columns,
-      columnCount,
-      currentPage,
-      hasError,
-      isSortedBy,
       loading,
-      pageSize,
+      hasError,
       rows,
+      columns,
       rowCount,
-      sortbyfield,
-      sortDesc,
-   } = useCsvapi(props.resource);
+      pageSize,
+      currentPage,
+      sortConfig,
+      isSortedBy,
+      sortByField,
+      changePage,
+      getTableInfos,
+    } = getDataTabularApi(props.resource)
 
-    apifyAndGetData();
+    getTableInfos(currentPage.value);
 
     return {
-      changePage,
-      columns,
-      columnCount,
-      currentPage,
-      hasError,
-      isSortedBy,
       loading,
-      pageSize,
+      hasError,
       rows,
+      columns,
       rowCount,
-      sortbyfield,
-      sortDesc,
+      pageSize,
+      currentPage,
+      sortConfig,
+      isSortedBy,
+      sortByField,
+      changePage,
     };
   }
 });
 </script>
+
+<style scoped>
+.style-cell{
+  height: 3rem;
+  overflow-y: auto;
+  display: block;
+  width: 100%;
+  font-size: 12px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+}
+.content-cell{
+  margin-top: auto;
+  margin-bottom: auto;
+}
+.col-width{
+  width: 20rem;
+}
+.cell-padding{
+  padding: 0.5rem 0rem 0.5rem 1.2rem!important
+}
+td{
+  border-right: 1px solid #CECECE;
+  border-bottom: 1px solid #CECECE;
+}
+th{
+  border-right: 1px solid #CECECE;
+}
+
+
+@media (min-width: 40rem){
```

Comment:
```suggestion
@media (min-width: 48em){
```

`48em` is the DSFR MD breakpoint